core/jvmTest/src/kotlinx/serialization/SerializationMethodInvocationOrderTest.kt

```diff
@@ -80,29 +80,39 @@ class SerializationMethodInvocationOrderTest {
 
         override fun <T : Any?> encodeSerializableValue(serializer: SerializationStrategy<T>, value: T) {
             when (step) {
-                0, 3 -> { step++; serializer.serialize(this, value); return }
+                0, 3 -> {
+                    step++; serializer.serialize(this, value); return
```

Comment:
If you want to make it multi-line, also split statements instead of using `;`

If it's autoformatting, better to revert it